projects/output-and-error-handling/client/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+    "bufio"
+    "fmt"
+    "net/http"
+	"os"
+	"time"
+	"strconv"
+	"errors"
+	"io"
+)
+
+func main() {
+    resp, err := http.Get("http://localhost:8080")
+
+   if err != nil {
+		if err != nil {
+		if errors.Is(err, io.EOF) {
+			fmt.Println("The server closed the connection unexpectedly. Please try again later.")
```

Comment:
Where is this going to print the message? Is that the right place for this kind of message?